.pre-commit-config.yaml

```diff
@@ -138,3 +138,15 @@ repos:
       - id: renovate-config-validator
         args:
           - --strict
+
+  - repo: local
+    hooks:
+      - id: codeowners-validator
+        name: CODEOWNERS validator
+        language: docker_image
+        pass_filenames: false
+        files: CODEOWNERS$
```

Comment:
I suppose technically this shouldn't be scoped only to the CODEOWNERS file, because if files or directories move, we need to re-run this validator.

```suggestion
```